### PR TITLE
Fix 'isNFT' flag initialization for assets

### PR DESCRIPTION
**Pull request by nickeskov**
This PR **breaks compatibility** with the previous state version.

### CODE REVIEW

File: pkg/state/assets.go [hunk 1]
```diff
@@ -23,70 +24,72 @@ type assetInfo struct {
...
 type assetConstInfo struct {
 	tail                 [proto.AssetIDTailSize]byte
 	issuer               crypto.PublicKey
 	decimals             uint8
 	issueHeight          proto.Height
+	isNFT                bool // isNFT flag must be initialized only once by calling assetInfo.initIsNFTFlag
 	issueSequenceInBlock uint32
 }
 
+type assetConstInfoRecord struct {
```
Comment (alexeykiselev) [inline pkg/state/assets.go:55]: Why keep both types `assetConstInfo` and `assetConstInfoRecord`? May be remove one? Or convert the old one in CBOR record?

File: pkg/state/fee_validation.go [hunk 2]
```diff
@@ -48,8 +48,8 @@ type feeValidationParams struct {
 }
 
 type assetParams struct {
-	quantity   int64
-	decimals   int32
+	quantity   uint64
```
Comment (alexeykiselev) [inline pkg/state/fee_validation.go:51]: I think that `int` types were chosen intentionally for better compatibility with Scala implementation.

File: pkg/state/fee_validation.go [hunk 3]
```diff
@@ -95,16 +95,16 @@ func minFeeInUnits(params *feeValidationParams, tx proto.Transaction) (uint64, e
 	fee := baseFee
 	switch txType {
 	case proto.IssueTransaction:
-		var asset assetParams
+		var ap assetParams
 		switch itx := tx.(type) {
 		case *proto.IssueWithSig:
-			asset = assetParams{int64(itx.Quantity), int32(itx.Decimals), itx.Reissuable}
```
Comment (alexeykiselev) [inline pkg/state/fee_validation.go:101]: May be better to add checks that values fit in `int` types here?

**Review by alexeykiselev — CHANGES_REQUESTED** [review]

**Review by alexeykiselev — APPROVED** [review]

**Comment by nickeskov** [comment]
Testnet import on commit [d4a85d0](https://github.com/wavesplatform/gowaves/pull/1333/commits/d4a85d0e6264d064fc98735209aaa94939391523) is ok

**Review by alexeykiselev — APPROVED** [review]

**Pull request merged**